guides/common/modules/proc_enabling-connections-to-capsule.adoc

```diff
@@ -13,17 +13,21 @@ include::snip_firewalld.adoc[]
 [options="nowrap"]
 ----
 # firewall-cmd \
---add-port="53/udp" --add-port="53/tcp" \
---add-port="67/udp" \
---add-port="69/udp" \
+--add-service=dns \
+--add-service=dhcp \
+--add-service=tftp \
 ifdef::katello,satellite,orcharhino[]
---add-port="80/tcp" --add-port="443/tcp" \
+--add-service=http \
+--add-service=https \
 --add-port="5647/tcp" \
 endif::[]
---add-port="8140/tcp" \
---add-port="8443/tcp" \
+--add-service=puppetmaster \
+ifndef::katello,satellite,orcharhino[]
+--add-service=foreman-proxy
+endif::[]
 ifdef::katello,satellite,orcharhino[]
---add-port="8000/tcp" --add-port="9090/tcp"
+--add-port="8000/tcp" \
+--add-port="9090/tcp"
```

Comment:
Can't we also add the `foreman-proxy` service here? Would that do something different for satellite than what it currently does?

Reply:
The firewalld service defines it as port 8443: https://github.com/firewalld/firewalld/blob/master/config/services/foreman-proxy.xml.

Today we discussed in our team, and there is a plan to move Katello's Smart Proxies to port 8443.

We used to run Candlepin (as a reverse proxy) on port 8443 on content proxies. Right now we're at the point where it has moved to port 443, but we maintain it on upgraded installations to allow users to migrate. We're close to the point where we want to drop that compatibility, which would free up the port. Then we can start to work on the Smart Proxy port migration as well. Once we've completed that, the ecosystem is standardized and we could use the port.

It is possible to modify the service in firewalld, but then the user no longer gets updates if firewalld changes the original service, so I opted to use the port number instead.

Reply:
Understood, thanks for explaining.